src/MobileAPIServiceProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace ColoredCow\LaravelMobileAPI;
+
+use ColoredCow\LaravelMobileAPI\Commands\SetupMobileAPI;
+use Illuminate\Support\ServiceProvider;
+use Laravel\Passport\Passport;
+
+class MobileAPIServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        Passport::routes();
+        $this->registerCommands();
+    }
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+
+    public function registerCommands()
+    {
+        if ($this->app->runningInConsole()) {
+            $this->commands([
+                SetupMobileAPI::class,
```

Comment:
Can we remove this command registration from here? Let's user decide what type of guard and token they want to implement.

Reply:
yes making sense.

@pankaj-ag just added this command so that deployment of the passport can be done without showing  `php artisan passport:install` command.